Projects/2-Intermediate/MCP-Server.md

```
@@ -0,0 +1,63 @@
+# Application Name
+
+**Tier:** 2-Intermediate
+
+A simple **Model Context Protocol (MCP) Server** that implements a part of the **MCP RPC Protocol** and integrates with **Claude Desktop**.  
+The goal of this project is to create a small automation tool that communicates with Claude through the MCP server interface.  
+The server exposes one or more tool that automate a task (e.g., tasks like file lookup, note retrieval, or text formatting).
+
+### Purpose
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)